src/main/java/org/gridsuite/study/server/dto/ComputationType.java

```diff
@@ -1,16 +1,10 @@
 package org.gridsuite.study.server.dto;
 
 public enum ComputationType {
-    LOAD_FLOW("LoadFlow"),
-    SECURITY_ANALYSIS("SecurityAnalysis"),
-    SENSITIVITY_ANALYSIS("SensitivityAnalysis"),
-    SHORT_CIRCUIT("ShortCircuitAnalysis"),
-    VOLTAGE_INITIALIZATION(""),
-    DYNAMIC_SIMULATION("");
```

Comment:
Maybe it's useful for deserialization in SupervisionController ? Not sure though

Reply:
I think it's safe to remove the "subReporterKey" here